### PR TITLE
Adds Bépo french layout

### PR DESCRIPTION
**Pull request by jchatard**
This PR introduces the French Bépo layout, which is gaining popularity.

Website: https://bepo.fr/wiki/Accueil
Wikipedia: https://en.wikipedia.org/wiki/B%C3%89PO

I hope I didn't make any mistakes! I don't have a Dygma keyboard yet, so I haven't been able to test it myself. :-)

### CODE REVIEW

**Comment by jchatard** [comment]
I just updated the PR with feedbacks provided by jreklund from Discord.

Ready to merge!

Thanks

File: src/api/keymap/languages/french/frenchBepo.js [hunk 1]
```diff
@@ -0,0 +1,1284 @@
...
+      code: OFFSET_SHIFT_ALTGR + 55,
+      labels: {
+        primary: "‡",
+      },
+      newGroupName: "Letters",
+    },
+    {
+      code: OFFSET_SHIFT_ALTGR + 56,
+      labels: {
+        primary: "⛄",
```
Comment (jreklund) [inline src/api/keymap/languages/french/frenchBepo.js:1171]: Snowman and other icons is still in this file.

File: src/api/keymap/languages/french/frenchBepo.js [hunk 2]
```diff
@@ -0,0 +1,1284 @@
...
+    },
+  },
+  {
+    code: 46,
+    labels: {
+      primary: "%",
+    },
+  },
+];
+const frenchLetters = [
```
Comment (jreklund) [inline src/api/keymap/languages/french/frenchBepo.js:110]: Should be a space (enter) just above these const.

File: src/api/keymap/db.js [hunk 3]
```diff
@@ -152,6 +156,7 @@ const languagesDB = {
   spanish,
   german,
   french,
+  frenchBepo: frenchBepo,
```
Comment (jreklund) [inline src/api/keymap/db.js:159]: Don't know if this mess something up, probably best to have the same format.

File: src/api/keymap/languages/french/frenchBepo.js [hunk 4]
```diff
@@ -0,0 +1,1284 @@
...
+    },
+    {
+      code: OFFSET_SHIFT_ALTGR + 49,
+      labels: {
+        primary: "🄯",
+      },
+      newGroupName: "Letters",
+    },
+    // Third row
+    // {
```
Comment (jreklund) [inline src/api/keymap/languages/french/frenchBepo.js:1103]: Unused code.

**Review by jreklund — COMMENTED** [review]
I wanted to check how the PR was going and saw that you deleted the icons from the overview. But the snowman and other icons is still in `src/api/keymap/languages/french/frenchBepo.js`, so it will render on the actual key when you have selected correct modifiers.

**Comment by javierguzman** [comment]
Thanks @jchatard we'll review it!

File: src/renderer/modules/Settings/GeneralSettings.js [hunk 5]
```diff
@@ -72,6 +72,7 @@ export default class GeneralSettings extends Component {
       spanishF,
       germanF,
       frenchF,
+      frenchF,
```
Comment (jytou) [inline src/renderer/modules/Settings/GeneralSettings.js:75]: Shouldn’t it be something like frenchBepo instead ?

Comment (jchatard) [reply src/renderer/modules/Settings/GeneralSettings.js:75]: It does not matter, the flag is the same, the French one. So… If we can avoid duplication.

**Comment by AlexDygma** [comment]
Thanks for your contribution @jchatard ! tested it and works 👌

**Pull request merged**